packages/flutter/test/material/menu_anchor_test.dart

```diff
@@ -3960,6 +3960,65 @@ void main() {
         tester.getRect(find.byKey(contentKey)).left + horizontalOffset,
       );
     });
+
+    group('The menu is attached to the bottom of the MenuAnchor content', () {
+      const double contentHeight = 100.0;
+      final MenuController menuController = MenuController();
+      final UniqueKey contentKey = UniqueKey();
+
+      Finder findMenuPanel() {
+        return find.byWidgetPredicate((Widget w) => '${w.runtimeType}' == '_MenuPanel');
+      }
+
+      MenuAnchor buildMenuAnchor() {
+        return MenuAnchor(
+          controller: menuController,
+          builderAlignment: AlignmentDirectional.topStart,
+          menuChildren: <Widget>[
+            MenuItemButton(
+              onPressed: () {},
+              child: const Text('Item 1'),
+            ),
+          ],
+          builder: (BuildContext context, MenuController controller, Widget? child) {
+            return SizedBox(key: contentKey, width: 100.0, height: contentHeight);
+          },
+          child: Container(width: 200, height: 60, color: Colors.purple),
```

Comment:
Child wouldn't be applicable in this scenario, since it's not being passed to the builder.

Reply:
Oh, good catch, that was a left over of an experiment i did.